reposilite-site/data/guides/migration/artifactory.md

```diff
@@ -0,0 +1,38 @@
+---
+id: artifactory
+title: Artifactory
+---
+
+### Instructions
+
+1. Export Each Repository from Artifactory.
+    Be sure to check the `.m2 compatible export` box to get an export that reposilite can use. Also, we recommend excluding artifactory build metadata if your version of Artifactory has this feature.
+2. Move the resulting export files to your new reposilite server and extract them to the repository folder you would like them in.
+    Depending on the organization of your new repositories, you may need to move your artifacts out of parent folders to get them in the location you want them in your new repository.
```

Comment:
Below the 2nd paragraph I'd show the directory structure in Reposilite, so it'll be more clear for a newbie users how it looks like. You can link this guide: https://reposilite-nl595cx48-dzikoysk.vercel.app/guide/manual#data-structure 
If you'll do this like that:

```
Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
Lorem Ipsum has been the industry's standard dummy text ever since the 1500s

* Link

Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
Lorem Ipsum has been the industry's standard dummy text ever since the 1500s
```

It'll also look like a natural spacer between paragraphs